### PR TITLE
Normalize paths of dependencies

### PR DESCRIPTION
**Pull request by strayiker**
This fixes bug where webpack ignores the incremental build and recompile all files in project (win 8.1)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ module.exports = function (content) {
             return result.css.toString();
         } catch (err) {
             formatSassError(err);
-            err.file && this.dependency(err.file);
+            err.file && addNormalizedDependency(err.file);
```
Comment (jhnns) [inline index.js]: I don't think that this is necessary. `formatSassError` [is already normalizing it](https://github.com/strayiker/sass-loader/blob/c51fff0c25d633b4df010e6ba3a95027d43c04ad/index.js#L55).

File: index.js [hunk 2]
```diff
@@ -160,7 +160,7 @@ module.exports = function (content) {
             }
             // Add the resolvedFilename as dependency. Although we're also using stats.includedFiles, this might come
             // in handy when an error occurs. In this case, we don't get stats.includedFiles from node-sass.
-            self.dependency(resolvedFilename);
+            addNormalizedDependency(resolvedFilename);
```
Comment (jhnns) [inline index.js]: webpack returns normalized paths afaik

**Comment by jhnns** [comment]
Could you give me an example where the problem occurs? The paths are already normalized afaik.

**Comment by strayiker** [comment]
https://github.com/strayiker/sass-loader-dependencies-bug

**Comment by jhnns** [comment]
Shipped with 2.0.1. Thanks! :+1:

**Pull request merged**